src/components/sidecar/get-truncated-title.test.ts

```diff
@@ -0,0 +1,45 @@
+import getTruncatedTitle from './get-truncated-title'
+
+describe('getTruncatedTitle', () => {
+  it('leaves a short title unmodified', () => {
+    const title = 'Hello world'
+    expect(getTruncatedTitle(title)).toBe(title)
+  })
+
+  it('does not add ... to a title of exact limit length', () => {
```

Comment:
I split this out and added a test file because I noticed the very specific `contrib Tools and Helpers...` issue, and wanted to debug.